jhove-modules/pdf-hul/src/main/resources/edu/harvard/hul/ois/jhove/module/pdf/ErrorMessages.properties

```diff
@@ -153,3 +153,4 @@ PDF-HUL-147 = Page tree node not found.
 PDF-HUL-148 = PDF minor version number is greater than 7.
 PDF-HUL-149 = Invalid indirect destination - referenced object ''{0}'' cannot be found
 PDF-HUL-150 = Cross-reference stream must be a stream
+PDF-HUL-151 = An error occurred reading the cross-reference
```

Comment:
"**Unexpected** error occurred while attempting to read the cross-reference **table**" might be more accurate.

Reply:
Thanks @david-russo . I updated the pull request. I hope I did it correctly.

Reply:
Yep, perfect.